bugbug/tools/code_review.py

```diff
@@ -1262,11 +1301,17 @@ def _generate_suggestions(self, patch: Patch):
                     },
                 )
 
+            must_created_before = patch.date_created if self.is_experiment_env else None
```

Comment:
```suggestion
            created_before = patch.date_created if self.is_experiment_env else None
```

Reply:
Dome in f2c3fafabbf644c72fd867689c6d025c42a31d33